docs/source/examples/flow_past_cylinder.rst

```diff
@@ -0,0 +1,252 @@
+.. _flow_past_cylinder:
+
+Flow past a circular cylinder using open boundary conditions
+----------------------------------------------
+
+This example demonstrates the API of inlet and outlet boundary conditions in PySPH. The flow past a circular cylinder is one the example which uses both inlet and outlet boundary condition.
+To run it one may do::
+
+  $ cd ~/pysph/pysph/examples/
+  $ python flow_past_cylinder_2d.py
+
+There are many command line options that this example provides, check them out with::
+
+  $ python flow_past_cylinder_2d.py -h
+
+The example source can be seen at `flow_past_cylinder_2d.py
+<https://github.com/pypr/pysph/tree/master/pysph/examples/flow_past_cylinder_2d.py>`_.
+
+
```

Comment:
Maybe describe the problem and the boundary conditions here so it is easier to understand?

Reply:
done